packages/cashier/src/Components/usdt-side-note.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { Modal } from '@deriv/components';
+import { Modal, Text } from '@deriv/components';
 import { Localize } from '@deriv/translations';
 
-const USDTSideNote = () => {
+const USDTSideNote = props => {
```

Comment:
Just one small suggestion: can we breakdown `type` here and remove `props.` in front of each variable call

Reply:
```suggestion
const USDTSideNote = { type } => {
```